generators/server/templates/src/main/java/package/_Application.java

```diff
@@ -54,6 +55,8 @@
     @Inject
     private Environment env;
 
+    public static ApplicationContext context;
```

Comment:
I thought app servers were the past ! :smile: 
My actual need is to access the app context from outside the app (from the remore shell in REPL mode). Do you know another way to get it ?